src/EditorFeatures/CSharpTest/SplitStringLiteral/SplitStringLiteralCommandHandlerTests.cs

```diff
@@ -744,5 +752,56 @@ static void Main(string[] args)
 }",
             verifyUndo: false);
         }
+
+        [WorkItem(20258, "https://github.com/dotnet/roslyn/issues/39040")]
+        [WpfFact, Trait(Traits.Feature, Traits.Features.SplitStringLiteral)]
+        public void TestMultiCaretSingleLine()
+        {
+            TestHandled(
+@"class C
+{
+    void M()
+    {
+        var v = ""now is [||]the ti[||]me"";
```

Comment:
=-o

I mean, it's cool this is supported!  but i would be 100% ok with us not doing anything in this case and just passing the `return` on untouched.

Reply:
i'm ok with this now.